packages/components/chip/src/variants/dashed.ts

```diff
@@ -0,0 +1,84 @@
+import { tw } from '@spark-ui/internal-utils'
+
+export const dashedVariants = [
+  {
+    intent: 'primary',
+    design: 'dashed',
+    class: tw([
+      'enabled:hover:bg-primary/dim-5',
```

Comment:
💅 , just a reminder that when using the `enabled` pseudo-class, these styles won't be applied if someone were to use the `asChild` pattern with an element that is not a button

Reply:
for sure, and if they don't add an onClick prop the default element is a button and this styles are never applied, (by default)